server/app.js

```diff
@@ -5,8 +5,8 @@ const application = express();
 const fs = require("fs");
 const formidable = require("formidable");
 const tesseract = require("node-tesseract");
-const LanguageTranslatorV3 = require("watson-developer-cloud/language-translator/v3");
-const NaturalLanguageUnderstandingV1 = require('watson-developer-cloud/natural-language-understanding/v1.js');
```

Comment:
Hmm, i'm not sure if the version parameter is OK to pass through, maybe it's just ignored?

https://github.com/watson-developer-cloud/node-sdk/releases

```
natural-language-understanding: The version_date parameter in Natural Language Understanding is no longer supported.
```

See line 17 below

Reply:
Hey @stevemart, It is mentioned that **version_date** is no longer supported but use **version** instead to migrate. 
<img width="893" alt="Screenshot 2019-05-03 at 12 03 45 PM" src="https://user-images.githubusercontent.com/11567379/57122538-89c10e80-6d9b-11e9-8f01-aa15e1981398.png">

I have used the **version** parameter in the code.
```
let natural_language_understanding = new NaturalLanguageUnderstandingV1({
  version: '2018-03-16'
});
```